test/cpp2rust/path with space/lib3.cpp

```diff
@@ -0,0 +1,10 @@
+// Check that libraries located at a path containing a space can also be linked.
```

Comment:
I'd go with a even more complex path for the test. Include a few symbol, some emoji, an such.

Reply:
I added some chinese characters and emojis. I don't really want to add any of the ascii special characters, because then we would have to check what the common subset of supported special characters in filepath is across our supported operating systems and generators/compilers.

Reply:
Looks like ninja 1.11 (which is not even out yet) would be required to support non-ansi filepaths on windows. See https://github.com/ninja-build/ninja/pull/1915 for details.